### PR TITLE
Browser extensions sidebar update - API differences article

### PR DESCRIPTION
**Pull request by rebloor**
Adding "The differences between extensions API implementations" to the porting section.

### CODE REVIEW

File: macros/AddonSidebar.ejs [hunk 1]
```diff
@@ -58,6 +58,7 @@ var text = mdn.localStringMap({
     'WebExtensions/Embedded_WebExtensions': 'Embedded WebExtensions',
     'WebExtensions/Comparison_with_the_Add-on_SDK': 'Comparison with the Add-on SDK',
     'WebExtensions/Comparison_with_XUL_XPCOM_extensions': 'Comparison with XUL/XPCOM extensions',
+    'WebExtensions/differences_between_extensions_API_implementations': 'Extension API differences',
```
Comment (ExE-Boss) [inline macros/AddonSidebar.ejs]: This should be <q>Differences between API implementations</q> located at [WebExtensions/differences_between_API_implementations](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/differences_between_API_implementations).

Comment (rebloor) [reply macros/AddonSidebar.ejs]: Updated

Comment (ExE-Boss) [reply macros/AddonSidebar.ejs]: The URL still points to the old location.

Comment (rebloor) [reply macros/AddonSidebar.ejs]: Sorry, URL fixed

**Comment by rebloor** [comment]
@wbamberg Thanks changes, hopefully, made as needed. This article is intended for the new extension workshop site, so is separate from the existing one.

**Pull request merged**